tutorials/upload-your-first-file-with-ethstorage-sdk.md

```diff
@@ -43,31 +43,9 @@ You will get a FlatDirectory address which is something like 0x...
 
 In this section, you will upload some files into the FlatDirectory that you just created.
 
-#### Browser
-
-On the web page, you can use the input tag to get the file.
+Callback
```

Comment:
The callback function is less important, and I think it will be better to put it on the end of this section